examples/example-tutorial-todo-app/src/mocks.ts

```diff
@@ -18,7 +18,7 @@ const projects: MockProject[] = [
     $apiName: "MockProject",
     $primaryKey: "1",
     id: "1",
-    name: "Fake Project",
+    name: "Mock Project",
```

Comment:
For casing consistency with `Yet another mock project`
```suggestion
    name: "Mock project",
```